src/DatabaseGpt.Npgsql/NpgsqlDatabaseGptProvider.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Data;
+using System.Text;
+using Dapper;
+using DatabaseGpt.Abstractions;
+using DatabaseGpt.Abstractions.Exceptions;
+using Npgsql;
+
+namespace DatabaseGpt.Npgsql;
+
+public class NpgsqlDatabaseGptProvider : IDatabaseGptProvider, IDisposable
+{
+    private readonly NpgsqlDatabaseGptProviderConfiguration settings;
+    private bool disposedValue;
+
+    public NpgsqlDatabaseGptProvider(NpgsqlDatabaseGptProviderConfiguration settings)
+    {
+        this.settings = settings;
+        this.connection = new NpgsqlConnection(settings.ConnectionString);
+        connection.Open();
+    }
+
+    private readonly NpgsqlConnection connection;
+
+    public string Name => "Postgres";
+    public string Language => "Postgres SQL";
+
+    public async Task<IEnumerable<string>> GetTablesAsync(IEnumerable<string> excludedTables)
+    {
+        var tables = await connection.QueryAsync<string>("""
+            SELECT TABLE_SCHEMA || '.' || TABLE_NAME AS Tables
+            FROM INFORMATION_SCHEMA.TABLES
+            WHERE TABLE_SCHEMA NOT IN ('pg_catalog', 'information_schema')
+            """);
+        return tables.Where(t => !excludedTables.Contains(t));
```

Comment:
Add a blank line before `return` statement. Add the support for included tables, as in https://github.com/marcominerva/DatabaseGPT/blob/master/src/DatabaseGpt/DatabaseGptClient.cs#L117-L131.